subworkflows/nf-core/mmseqs_contig_taxonomy/meta.yml

```diff
@@ -0,0 +1,57 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/subworkflows/yaml-schema.json
+name: "mmseqs_contig_taxonomy"
+description: Assign taxonomy to contigs using the MMseqs2 workflow.
+keywords:
+  - metagenomics
+  - database
+  - contigs
+  - mmsesq2
+  - taxonomy
+components:
+  - mmseqs/databases
+  - mmseqs/createdb
+  - mmseqs/taxonomy
+  - mmseqs/createtsv
+input:
+  - ch_contigs:
+      type: file
+      description: |
+        Channel containing each fasta in nucleotide format as a distinct element with meta.
+        Structure: [ val(meta), path(fasta) ]
+      pattern: "*.{fasta,fa,fna}"
+  - ch_mmseqs_databases_localpath:
+      type: string
+      description: |
+        Channel containing a database created by mmseqs2 databases.
+        Structure: [ path(mmseqsdb) ]
+      pattern: "*/mmseqs/database"
+  - ch_mmseqs_databases_id:
+      type: string
+      description: |
+        Channel containing the ID of a database made available by developers of mmseqs2. Please refer to https://github.com/soedinglab/MMseqs2/wiki#downloading-databases for possible IDs to use.
+        Structure: [ val(ID) ]
+      pattern: "SILVA"
```

Comment:
might be me, but could this be a copy paste mistake?

Reply:
If you are pointing to the pattern, no it can be kalamari or SILVA or other IDS listed in their guideline.

Reply:
but then the pattern isn't limited to just "SILVA" right? If not, i would just either remove the pattern and if the linter complains write something as "[A-z]". What do you think?

Reply:
well i do explain it in the description so thats clear eneough i guess. I removed it now and the the linter is happy ;)